src/App.css

```diff
@@ -0,0 +1,92 @@
+.App {
```

Comment:
all css files should be completely deleted....you should be doing your styles using material ui

Reply:
doing in separate branch

Reply:
doing in separate branch